### PR TITLE
ecdfplot stat parameter manage percent

### PR DESCRIPTION
**Pull request by gregorystrubel**
I propose this PR to solve #3334

### CODE REVIEW

File: seaborn/_statistics.py [hunk 1]
```diff
@@ -405,13 +405,13 @@ def __init__(self, stat="proportion", complementary=False):
 
         Parameters
         ----------
-        stat : {{"proportion", "count"}}
+        stat : {{"percent", "proportion", "count"}}
```
Comment (mwaskom) [inline seaborn/_statistics.py:408]: Please keep the default value first in the list here

File: tests/test_statistics.py [hunk 2]
```diff
@@ -436,6 +436,15 @@ def test_univariate_count(self, x):
...
 
+        assert_array_equal(vals[1:], np.sort(x))
+        assert_array_almost_equal(stat[1:], np.linspace(0, 100, len(x) + 1)[1:])
+        assert stat[0] == 0
+
+    def test_univariate_percent(self, x):
+
+        ecdf = ECDF(stat="percent")
+        stat, vals = ecdf(x)
+
```
Comment (mwaskom) [inline tests/test_statistics.py:447]: This diff changes the assertion for the count test and then adds a new setup for the percent test but uses the old count test assertion. I think it's only passing by accident because there are 100 entries in the test data. (Probably that is a bad idea and a less-round number should be used for the data fixture to avoid edge cases like this).

File: doc/_docstrings/ecdfplot.ipynb [hunk 3]
```diff
@@ -94,6 +94,15 @@
...
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sns.ecdfplot(data=penguins, x=\"bill_length_mm\", hue=\"species\", stat=\"percent\")"
+   ]
+  },
```
Comment (mwaskom) [inline doc/_docstrings/ecdfplot.ipynb:105]: I don't think we need a separate example here.

**Review by mwaskom — CHANGES_REQUESTED** [review]
Thanks, I'm 👍 on the new feature but have a couple of documentation nitpicks and think the test is wrong.

**Comment by gregorystrubel** [comment]
Thanks a lot for your review.
I've implemented all your comments.

**Comment by gregorystrubel** [comment]
I've reverted the revert concerning (mentioning that counts or percents can be shown).
Thanks a lot

**Review by mwaskom — CHANGES_REQUESTED** [review]

File: tests/test_statistics.py [hunk 4]
```diff
@@ -437,7 +441,16 @@ def test_univariate_count(self, x):
         stat, vals = ecdf(x)
 
         assert_array_equal(vals[1:], np.sort(x))
-        assert_array_almost_equal(stat[1:], np.arange(len(x)) + 1)
+        assert_array_almost_equal(stat[1:], np.linspace(0, 100, len(x) + 1)[1:])
```
[the remark this answers is not in the document]
Comment (gregorystrubel) [reply tests/test_statistics.py]: It was an error sorry to have miss it

File: tests/test_statistics.py [hunk 5]
```diff
@@ -437,7 +441,16 @@ def test_univariate_count(self, x):
...
+        assert stat[0] == 0
+
+    def test_univariate_percent(self, x2):
+
+        ecdf = ECDF(stat="percent")
+        stat, vals = ecdf(x2)
+
+        assert_array_equal(vals[1:], np.sort(x2))
+        assert_array_almost_equal(stat[1:], (np.arange(len(x2)) + 1)/len(x2)*100)
```
[the remark this answers is not in the document]
Comment (gregorystrubel) [reply tests/test_statistics.py]: corrected

**Comment by gregorystrubel** [comment]
Thanks a lot for your review.
I've implemented all your comments.

File: tests/test_statistics.py [hunk 6]
```diff
@@ -440,6 +444,15 @@ def test_univariate_count(self, x):
         assert_array_almost_equal(stat[1:], np.arange(len(x)) + 1)
         assert stat[0] == 0
 
+    def test_univariate_percent(self, x2):
+
+        ecdf = ECDF(stat="percent")
+        stat, vals = ecdf(x2)
+
+        assert_array_equal(vals[1:], np.sort(x2))
+        assert_array_almost_equal(stat[1:], (np.arange(len(x2))+1)/len(x2)*100)
```
Comment (mwaskom) [inline tests/test_statistics.py:453]:
```suggestion
        assert_array_almost_equal(stat[1:], np.arange(len(x2) + 1) / len(x2) * 100)
```

File: tests/test_statistics.py [hunk 7]
```diff
@@ -440,6 +444,15 @@ def test_univariate_count(self, x):
         assert_array_almost_equal(stat[1:], np.arange(len(x)) + 1)
         assert stat[0] == 0
 
+    def test_univariate_percent(self, x2):
+
+        ecdf = ECDF(stat="percent")
+        stat, vals = ecdf(x2)
+
+        assert_array_equal(vals[1:], np.sort(x2))
+        assert_array_almost_equal(stat[1:], np.arange(len(x2) + 1) / len(x2) * 100)
```
Comment (mwaskom) [inline tests/test_statistics.py:453]:
```suggestion
        assert_array_almost_equal(stat[1:], (np.arange(len(x2)) + 1) / len(x2) * 100)
```

**Comment by mwaskom** [comment]
Thanks @gregorystrubel

**Comment by gregorystrubel** [comment]
> Thanks @gregorystrubel

My pleasure
